services/hype-backend/config/auth.ts

```diff
@@ -2,6 +2,8 @@ import { registerAs } from '@nestjs/config';
 
 export default registerAs('auth', () => ({
   walletEndpoint: process.env.WALLET_HOST || 'http://localhost:3006',
+  secret: process.env.AUTH_SECRET || 'secret',
+  tokenExpiry: process.env.JWT_EXPIRY_TIME || '30 days',

```

Comment:
```suggestion
  tokenExpiry: process.env.JWT_EXPIRY_TIME || '7 days',
```
The default value in the .env.example is set to 7, let's keep it consistent here too.